assets/src/app/layout/main/layout.tsx

```diff
@@ -1,11 +1,8 @@
-import { observer } from 'mobx-react-lite';
 import { Header } from '~/components/header';
 
-export const MainLayout = observer(({ children }: { children: any }) => {
-  return (
-    <div className='min-h-screen'>
-      <Header />
-      {children}
-    </div>
-  );
-});
+export const MainLayout = ({ children }: { children: any }) => {
+  return <div className="min-h-screen">
+    <Header />
+    {children}
+  </div>;
```

Comment:
needs () brackets in return

Reply:
fix 23f5a1196f2889035990d78c68d4fbfabc1c1df6